readthedocsext/theme/templates/projects/version_form.html

```diff
@@ -53,7 +53,11 @@ <h2 class="ui medium dividing header">
           <div class="content">
             {% if version.active %}
               <div class="ui warning message">
-                {% trans "Deactivating a version or changing its slug will result in the deletion of its documentation." %}
+                {% url "projects_redirects_create" project_slug=project.slug as url_redirects %}
+                {% blocktrans trimmed with url=url_redirects %}
+                  Deactivating a version or changing its slug will result in the deletion of its documentation.
+                  Make sure to <a href="{{ url }}" target="_blank">create a redirect</a> to avoid not found pages.
```

Comment:
"not found pages" could be clearer language.

```suggestion
                  Make sure to <a href="{{ url }}" target="_blank">create a redirect</a> for these URLs to ensure old links continue working.
```